src/os/win/pa_win_wdmks_utils.c

```diff
@@ -25,28 +25,28 @@
  */
 
 /*
- * The text above constitutes the entire PortAudio license; however, 
+ * The text above constitutes the entire PortAudio license; however,
  * the PortAudio community also makes the following non-binding requests:
  *
  * Any person wishing to distribute modifications to the Software is
  * requested to send the modifications to the original developer so that
- * they can be incorporated into the canonical version. It is also 
- * requested that these non-binding requests be included along with the 
+ * they can be incorporated into the canonical version. It is also
+ * requested that these non-binding requests be included along with the
  * license above.
  */
 
 #include <windows.h>
 #include <mmreg.h>
 #ifndef WAVE_FORMAT_IEEE_FLOAT
     #define WAVE_FORMAT_IEEE_FLOAT 0x0003   // MinGW32 does not define this
-#endif    
+#endif
 #ifndef _WAVEFORMATEXTENSIBLE_
     #define _WAVEFORMATEXTENSIBLE_          // MinGW32 does not define this
 #endif
 #ifndef _INC_MMREG
     #define _INC_MMREG                      // for STATIC_KSDATAFORMAT_SUBTYPE_IEEE_FLOAT
 #endif
-#include <winioctl.h>						// MinGW32 does not define this automatically
+#include <winioctl.h>                        // MinGW32 does not define this automatically
```

Comment:
Tabs in the middle of a line should add a variable number of spaces to force tab alignment.